### PR TITLE
Fixing `ZeroDivisionError` thrown by UI sliders when the `value_range` is zero (0)

### PR DESCRIPTION
**Pull request by ganimtron-10**
Currently, the UI sliders throw `ZeroDivisionError` whenever the `value_range` is zero ie. min_value and max_value are the same.
Adding checks to avoid these errors and setting the handle at the start.

### CODE REVIEW

**Comment by m-agour** [comment]
Hello @ganimtron-10,
Thanks for making this PR, it's really important. Also, thanks for adding the unit test.

**Review by xtanion — COMMENTED** [review]
Hi @ganimtron-10, I tried this with [this](https://github.com/xtanion/fury/blob/gltf-anim-merge-kf/docs/tutorials/01_introductory/viz_simple_gltf_animation.py) animation example and the `Fox` model. It was causing `ZeroDivisionError` previously. It's working fine now 👍🏽.

I still need to check with different models and the code, I'll let you know if I find anything.

File: fury/ui/tests/test_elements.py [hunk 1]
```diff
@@ -472,6 +472,23 @@ def test_ui_range_slider(interactive=False):
...
+    sliders.append(ui.LineSlider2D(min_value=0, max_value=0, center=(200, 450)))
+    sliders.append(ui.LineDoubleSlider2D(min_value=0, max_value=0, center=(200, 200)))
+    sliders.append(ui.RingSlider2D(min_value=0, max_value=0, center=(200, 300)))
+    sliders.append(ui.RangeSlider(min_value=0, max_value=0))
+
+    current_size = (600, 600)
+    show_manager = window.ShowManager(size=current_size,
+                                      title="FURY Sliders Value Range Test")
+
+    show_manager.scene.add(*sliders)
```
Comment (skoudoro) [inline fury/ui/tests/test_elements.py:486]: what is the test here? I do not see any assertion. you need to improve these tests

**Review by skoudoro — COMMENTED** [review]
hi @ganimtron-10,

The fix looks good to me. however, the tests are weird.... Can you update your tests? thanks

**Comment by m-agour** [comment]
Hi @ganimtron-10 ,
Thank you for updating the tests. Can you also test for other range cases such as the following for each slider?
```
    with npt.assert_no_warnings():
        line_slider = ui.LineSlider2D(min_value=0, max_value=0,
                                      center=(200, 450))

        assert_equal(line_slider.value, 0)
        assert_equal(line_slider.min_value, 0)
        assert_equal(line_slider.max_value, 0)
        line_slider.value = 100
        assert_equal(line_slider.value, 0)
        line_slider.value = -100
        assert_equal(line_slider.value, 0)

        line_slider = ui.LineSlider2D(min_value=0, max_value=100,
                                      center=(200, 450))
        line_slider.value = 105
        assert_equal(line_slider.value, 100)
        line_slider.value = -100
        assert_equal(line_slider.value, 0)
```

**Comment by ganimtron-10** [comment]
Thank You @m-agour for the suggestion.
I have updated the tests!
PTAL.
Thanks!

**Review by skoudoro — APPROVED** [review]
LGTM, thanks @ganimtron-10 and @m-agour ! merging

**Pull request merged**